config/application.yml.default

```diff
@@ -593,6 +594,7 @@ test:
   skip_encryption_allowed_list: '[]'
   state_tracking_enabled: true
   team_ada_email: 'ada@example.com'
+  drop_off_report_config: '[{"email":"ursula@example.com","issuers":"urn:gov:gsa:openidconnect.profiles:sp:sso:agency_name:app_name"}]'
```

Comment:
duplicate key (and as i mentioned above, i think the email and issuers values should probably be a list)